cloudify_aws/ec2/resources/image.py

```diff
@@ -159,10 +159,11 @@ def create(ctx, iface, resource_config, **_):
                          resource_type=RESOURCE_TYPE,
                          waits_for_status=False)
 @decorators.wait_for_delete(status_deleted=['deregistered'])
-def delete(ctx, iface, resource_config, **_):
+def delete(ctx, iface, resource_config, dry_run=False, **_):
     """delete/deregister an AWS EC2 Image"""
+    resource_config['DryRun'] = dry_run
     if not ctx.instance.runtime_properties.get('use_external_resource'):
-        params = {'ImageId': iface.resource_id}
+        params = {'ImageId': iface.resource_id, 'DryRun': dry_run}
         try:
```

Comment:
We probably don't need this because it does not use the tagging decorator.
Also, why add DryRun twice?